demo/src/main/java/com/example/demo/dao/gamestate/GameStateDao.java

```diff
@@ -0,0 +1,14 @@
+package com.example.demo.dao.gamestate;
+
+import com.example.demo.models.dto.GameState;
+
+import java.util.List;
+
+public interface GameStateDao {
+    GameState getStateById(Long id);
+    GameState getStateByUserId(Long id);
+    GameState saveStateGame(GameState game);
+    List<GameState> getAll();
+    void setGame(Long idGame, Long idUserOne, Long idUserSecond, GameState game);
```

Comment:
should it be part of DAO ??

Reply:
if so it can be either saveGame (returning gameId or whole game object) and adduserToGame or smth like this